api/v1beta1/hcloudmachine_validation_test.go

```diff
@@ -130,7 +130,9 @@ func TestValidateHCloudMachineSpec(t *testing.T) {
 
 			if tt.want == nil {
 				assert.Empty(t, got)
-			} else if assert.Len(t, got, 1) {
+			} else if len(got) != 1 {
```

Comment:
are you sure you need all the else statements? I think you can actually remove all of them

Reply:
@yrs147 you haven't pushed anything since I reviewed it. I don't think this is resolved :sweat_smile: I'll unresolve it